resources/views/layouts/app.blade.php

```diff
@@ -11,87 +11,86 @@
 
     <!-- Fonts -->
     <link rel="dns-prefetch" href="//fonts.gstatic.com">
-    <link href="https://fonts.bunny.net/css?family=Nunito" rel="stylesheet">
```

Comment:
Denne fik skulle der kun være den ændring, som jeg kommenterer på her.
At den siger en masse andet, skyldes et af mine fuck-ups.
Jeg tog for lidt siden en rå kopi af filen fra develop, og der _skulle_ kun være den her linie, som er ændret